pkgs/applications/misc/gnuradio-ais/default.nix

```diff
@@ -12,8 +12,9 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "bistromath";
     repo = "gr-ais";
-    rev = "1863d1bf8a7709a8dfedb3ddb8e2b99112e7c872";
-    sha256 = "1vl3kk8xr2mh5lf31zdld7yzmwywqffffah8iblxdzblgsdwxfl6";
+    # Upstream PR: https://github.com/bistromath/gr-ais/commit/8502d0252a2a1a9b8d1a71795eaeb5d820684054
+    "rev" = "8502d0252a2a1a9b8d1a71795eaeb5d820684054";
```

Comment:
Don't forget to update "version" above.

Reply:
It seems weird that version goes back one year. Can you clarify a bit in the commit message what's going on here? Also, perhaps make a point of switching from building upstream branch to a PR.

Reply:
Not sure where the 2016-08-26 date even originated from as the commit referenced (https://github.com/bistromath/gr-ais/commit/1863d1bf8a7709a8dfedb3ddb8e2b99112e7c872) is on the date I changed it to. 
But yeah I'll add both for clarification